child_compassion/wizard/child_description_fr.py

```diff
@@ -174,14 +175,14 @@ def _get_guardians_info_fr(cls, cr, uid, child, case_study, context=None):
                        'stepfather', 'godfather']
         plur_values = ['friends', 'other relatives', 'foster parents']
         live_with = []
-        male_guardian = ''
-        female_guardian = ''
+        male_guardian = dict()
+        female_guardian = dict()
         for guardian in case_study.guardians_ids:
             value = (guardian.value_fr if guardian.value_fr
                      else guardian.value_en)
             if guardian.value_en in male_values:
                 live_with.append(u'son %s' % value)
-                male_guardian = value if not male_guardian else male_guardian
+                male_guardian = [guardian.value_en, value]
             elif guardian.value_en in plur_values:
                 live_with.append(u'des %s' % value)
             else:

```

Comment:
Add a comment telling that in this case (neither male nor plural guardian), the guardian is a female guardian. You can also remove the first test at line 189, because female_guardian is a dictionary and will never be equal to 'institutional_worker'.